services/frontend-v3/src/pages/ProfileEdit.jsx

```diff
@@ -1,25 +1,19 @@
 import React from "react";
+import { injectIntl } from "react-intl";
 import EditProfileLayout from "../components/layouts/editProfileLayout/EditProfileLayout";
 
-class ProfileCreate extends React.Component {
-  goto = (link) => this.props.history.push(link);
+const ProfileCreate = (props) => {
```

Comment:
Seems like it should be ProfileEdit instead of ProfileCreate, aside from that everything seems good

Reply:
done